Makefile

```diff
@@ -2,7 +2,7 @@ CXX = /usr/bin/g++
 NVCC = /usr/local/cuda/bin/nvcc
 CFLAGS = -O2
 LDFLAGS =
-LDLIBS = -lcuda -lcublas -lboost_regex -lboost_iostreams -lboost_program_options -lmsgpack
+LDLIBS = -lcuda -lcublas -lboost_regex -lboost_iostreams -lboost_program_options -lmsgpackc
```

Comment:
Since you added a new Makefile to be compatible with newer versions of libraries, this change to the original Makefile seems not particularly necessary at this time. We may merge two versions of Makefile later.

Reply:
Did the same modification to the original Makefile.